Pod/Swiftline/Args/Args.swift

```diff
@@ -29,20 +29,26 @@ public class Args {
       return result
     }
     
-    var result = [String: String]()
-    let parsedArges = ArgsParser.parseFlags(all)
+    var parsedFlags = [String: String]()
+    let parsedArgs = ArgsParser.parseFlags(all)
     
-    parsedArges.0.forEach {
-      result[$0.argument.name] = $0.value ?? ""
+    parsedArgs.0.forEach {
+      parsedFlags[$0.argument.name] = $0.value ?? ""
     }
-    
-    cachedResults = ParsedArgs(flags: result, parameters: parsedArges.1)
+
+    let commandName = parsedArgs.1.first!
+    var arguments = parsedArgs.1
```

Comment:
Ah, good suggestion about the ordering. but, I assumed that the force unwrap of `cachedResults` on line 43 made this one safe as well. Is that not a safe assumption? I think it is safe because it should *always* have at least one element, which would be the name of the executable.